src/App.js

```diff
@@ -1,26 +1,36 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import { Route, Switch } from 'react-router-dom'
+import List from './List.js'
+import AdminPage from './AdminPage.js'
+import HomePage from './HomePage.js'
+import Header from './Header.js'
+import DigimonDetail from './DigimonDetail.js'
+import './App.css'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default class App extends Component {
+    render() {
+        return (
+            <div className='container'>
+                <Header />
+                <Switch>
+                    <Route
+                    path="/home"
```

Comment:
If you make this path just a slash...'/'...then your homepage will load when the use hits the site. Currently, you have no links on the landing page and the use has to add /home to the url to get to your 'Home' page.